android/app/src/main/java/org/smartregister/fhircore/fragment/PatientDetailFragment.kt

```diff
@@ -140,6 +142,14 @@ class PatientDetailFragment : Fragment() {
               nextDoseNumber,
               nextVaccineDate
             )
+          val cal: Calendar = Calendar.getInstance()
+          cal.add(Calendar.DATE, -28)
+          val overDueStart = cal.time
+          if (immunization.recorded.before(overDueStart)) {
```

Comment:
Checking overdue vaccines may become a common operation, could we move it into a util or private method `isOverdue` that checks something like: 

```
immunization.recorded.add(28).before(Calendar.getInstance().time)
```

of course replacing the 28 with a function param.

Reply:
logic moved in Util.